Playlist_Genre_Country.php

```diff
@@ -115,48 +179,49 @@
 									ORDER BY Song_ID ASC");
 
 
-						//Runs and stores the query using the variable $con (see nav.php) and $query
-						$result = mysqli_query($conn,$query);
-						//runs in a 'while' loop
-						while($output=mysqli_fetch_array($result))
-						{
-						?>
-						<!--php is above. HTML is below. Used to output the query results -->
+								//Runs and stores the query using the variable $con (see nav.php) and $query
+								$result = mysqli_query($conn,$query);
+								//runs in a 'while' loop
+								while($output=mysqli_fetch_array($result))
+								{
+								?>
+								<!--php is above. HTML is below. Used to output the query results -->
 
 
-						<heading2>
-							<space></space>
+								<heading2>
+									<space></space>
 
-							<Song_ID2><p2 class = 'white'><?php echo $output['Song_ID']; ?></p2></Song_ID2>
-							<img src="images/<?php echo $output['Images']; ?>" style="width: 50px; height: 50px;">
-							<Title2><p2 class = 'white'><?php echo $output['Title']; ?></p2></Title2>
-							<Artist2><p2 class = 'white'><?php echo $output['Artist']; ?></p2></Artist2>
-							<Album2><p2 class = 'white'><?php echo $output['Album']; ?></p2></Album2>
-							<Genre2><p2 class = 'white'><?php echo $output['Genre']; ?></p2></Genre2>
-							<Size2><p2 class = 'white'><?php echo $output['Size']; ?></p2></Size2>
-							<Duration2><p2 class = 'white'><?php echo $output['Seconds']; ?></p2></Duration2>
-							<space></space>
+									<Song_ID2><p2 class = 'white'><?php echo $output['Song_ID']; ?></p2></Song_ID2>
+									<img src="images/<?php echo $output['Images']; ?>" style="width: 4vw; height: 4vw;">
```

Comment:
added a responsive code to make images smaller when screen is on a smaller device

Reply:
added to all my queries w/ images